includes/tasks/class-wc-calypso-task-appearance.php

```diff
@@ -0,0 +1,42 @@
+<?php
+
+namespace Automattic\WooCommerce\Admin\Features\OnboardingTasks\Tasks;
+
+use Automattic\WooCommerce\Admin\Features\OnboardingTasks\Tasks\Appearance;
+use Automattic\WooCommerce\Admin\Features\OnboardingTasks\Tasks\HeadstartProducts;
+
+/**
+ * WCBridgeAppearance Task
+ *
+ * @since   2.0.11
+ * @version 2.0.11
+ */
+class WCBridgeAppearance extends Appearance {
+	/**
+	 * Addtional data.
+	 *
+	 * @return array
+	 */
+	public function get_additional_data() {
+		return array(
+			'has_homepage' => self::has_homepage(),
+			'has_products' => HeadstartProducts::has_products(),
```

Comment:
I use `HeadstartProducts` here so we can get the correct products count to exclude sample products generated by Headstart themes